src/masternodes/rpc_customtx.cpp

```diff
@@ -195,6 +195,19 @@ class CCustomTxRpcVisitor
         rpcInfo.pushKV("source", obj.source.ToString());
         rpcInfo.pushKV("destination", std::to_string(obj.destination));
     }
+    
+
+    void operator()(const CLockDUSDMessage& obj) const {
+        CTxDestination dest;
+        if (ExtractDestination(obj.owner, dest)) {
+            rpcInfo.pushKV("owner", EncodeDestination(dest));
+        } else {
+            rpcInfo.pushKV("owner", "Invalid destination");
+        }
+
+        rpcInfo.pushKV("dusdIn", GetDecimaleString(obj.dusdIn));
+        rpcInfo.pushKV("lockTime", std::to_string(obj.lockTime));
```

Comment:
Don't convert it to string, just cast it to plain `int` as UniValue doesn't support `(u)intX_t` types